src/UI/Internal/RenderConfig.elm

```diff
@@ -0,0 +1,39 @@
+module UI.Internal.RenderConfig exposing (Locale(..), RenderConfig(..), RenderConfigData, string, strings)
+
+import Element
+import I18n.English as LangEnglish
+import I18n.Spanish as LangSpanish
+import I18n.Type as I18n
+
+
+type alias RenderConfigData =
+    { deviceClass : Element.DeviceClass
+    , deviceOrientation : Element.Orientation
+    , locale : Locale
+    }
+
+
+{-| `RenderConfig.RenderConfig` upholds all the information required for the components to apply the responsivity and accessibility changes.
+-}
+type RenderConfig
+    = RenderConfig RenderConfigData
+
+
+type Locale
```

Comment:
We do not have any distinction between south American Spanish and Spanish from Spain, same for English, so I would call it just as English | Spanish

Reply:
:thinking: we have one detail, our English use dates in the format "DD/MM/YYYY".

Reply:
Still don't think it's relevant enough to have this distinction here, once we have it, we do it, otherwise, let's use English and Spanish.